src/tasks.ts

```diff
@@ -3,92 +3,120 @@
 import * as vscode from 'vscode';
 import * as fs from 'fs';
 import * as path from 'path';
+import { makeTerminalOptions } from './rTerminal';
+
+
+const TYPE = 'R';
 
 interface RTaskDefinition extends vscode.TaskDefinition {
-    code: string[];
-    cwd?: string;
-    env?: { [key: string]: string };
+    type: string,
+    code: string[],
+    args?: string[],
+    cwd?: string,
+    env?: { [key: string]: string }
 }
 
+interface RTaskMeta {
+    definition: RTaskDefinition,
+    problemMatchers?: string | string[],
+    name?: string,
+    group?: vscode.TaskGroup
+}
 
-const TYPE = 'R';
 
-class TaskMeta {
-    constructor(
-        public name: string,
-        public definition: RTaskDefinition,
-        public group?: vscode.TaskGroup,
-        public problemMatchers?: string | string[],
-    ) {}
-}
+const getRterm = async function ():Promise<string> {
+    const termOptions = await makeTerminalOptions();
```

Comment:
We should not use `r.rterm.*` here, it is used for interactive terminal (e.g. radian). We should only use `r.rpath.*` in tasks.